packages/components/src/utils/validation.ts

```diff
@@ -38,6 +38,20 @@ const isValidOneMapEmbedUrl = (urlObject: URL) => {
   )
 }
 
+// Slug pattern obtained from:
+// https://github.com/opengovsg/maps/blob/8c3f2e6c9af1be78fac5fc69115858241ae1f720/apps/web/modules/map/schema.ts#L3-L16
```

Comment:
can update to `develop` instead of `8c3f2e6c9af1be78fac5fc69115858241ae1f720`?

Reply:
Removed, but we should reference the actual commit hash because the line numbers can run if they make changes to the file in the future